PSSwagger.LiveTestFramework/build/PSSwagger.LiveTestFramework.Build.psm1

```diff
@@ -0,0 +1,338 @@
+        if ((Get-Content -Path $_.FullName | Out-String).Contains("<Project Sdk=`"Microsoft.NET.Sdk`"")) {
+            $cache = @{}
+            Build-DotNetProject -Project $_ -Framework $Framework -Configuration $Configuration `
+                               -OutputDirectory $OutputDirectory `
+                               -ProjectCache $cache `
+                               -Clean -Publish
+            if ($OutputDirectory) {
+                $src = (Join-Path -Path $_.DirectoryName -ChildPath "bin" | 
+                    Join-Path -ChildPath $Configuration |
+                    Join-Path -ChildPath $Framework |
+                    Join-Path -ChildPath "publish" |
+                    Join-Path -ChildPath "*")
+                Write-Verbose -Message "Copying files from '$src' to '$OutputDirectory'"
+                Copy-Item -Path $src -Destination $OutputDirectory
+            }
+        }
+    }
+}
+
+function Build-DotNetProject {
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory=$false)]
+        [System.IO.FileInfo]
+        $Project,
+
+        [Parameter(Mandatory=$false)]
+        [string]
+        [ValidateSet('net452')]
+        $Framework = 'net452',
+
+        [Parameter(Mandatory=$false)]
+        [string]
+        [ValidateSet('Debug','Release')]
+        $Configuration = 'Debug',
+
+        [Parameter(Mandatory=$false)]
+        [string]
+        $OutputDirectory,
+
+        [Parameter(Mandatory=$false)]
+        [switch]
+        $Clean,
+
+        [Parameter(Mandatory=$false)]
+        [switch]
+        $Publish,
+
+        [Parameter(Mandatory=$false)]
+        [switch]
+        $Test,
+
+        [Parameter(Mandatory=$false)]
+        [string]
+        $TestLogger,
+
+        [Parameter(Mandatory=$false)]
+        [hashtable]
+        $ProjectCache
+    )
+
+    Write-Verbose -Message "Building project: $Project"
+    pushd $Project.DirectoryName
+    if ($ProjectCache) {
+        if ($ProjectCache.ContainsKey($Project.FullName)) {
+            return
+        }
+
+        $projectRefs = Select-Xml -Path $Project.FullName -XPath "/Project/ItemGroup/ProjectReference" | 
+            Select-Object -ExpandProperty Node |
+            Select-Object -ExpandProperty Include |
+            Resolve-Path
+        foreach ($projectRef in $projectRefs) {
+            $ProjectCache[$projectRef] = $true
+        }
+    }
+
+    dotnet clean
+    dotnet restore
+    dotnet build --framework $Framework
+    if ($Publish) {
+        dotnet publish --framework $Framework
+    }
+    if ($Test) {
+        if ($TestLogger) {
+            dotnet test --framework $Framework --logger $TestLogger
+        } else {
+            dotnet test --framework $Framework
+        }
+    }
+    popd
+}
+
+function Setup-PSSwaggerUtility {
+    [CmdletBinding()]
+    param()
+
+    if (-not (Get-Module -Name PSSwaggerUtility -ListAvailable)) {
+        $p = Find-Package -Name PSSwaggerUtility
```

Comment:
>Find-Package -Name PSSwaggerUtility [](start = 13, length = 35)

Consider using PowerShellGet cmdlets with explicit source/repo name.
or specify the provider name and source name for *-Package cmdlets.